.github/workflows/build-lint-test.yml

```diff
@@ -75,21 +75,17 @@ jobs:
         with:
           go-version-file: go.mod
 
+      - name: Install golangci-lint
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/HEAD/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.1.6
```

Comment:
Would downloading the executable directly be an option, similar to https://github.com/golangci/golangci-lint-action/blob/main/src/install.ts#L152? I am unsure about running a script with `HEAD`.

Reply:
OR could we even use the official GitHub action and pass it the command line args?

Reply:
I'll go with the second one 👍